packages/dep-tree-js/index.js

```diff
@@ -3,22 +3,19 @@ const path = require("path");
 const fs = require("fs");
 const builtInPackages = require("module").builtinModules || [];
 
+const avaibleExtensions = [".js", ".jsx", ".md", ".mdx", ".ts", ".tsx", ".vue"];
```

Comment:
@aykutkardas can you please change this variable's name to `availableExtensions`.

Reply:
Oh sorry.

Reply:
I sent it along with a change. @asadm